src/App.css

```diff
@@ -1,38 +1,134 @@
+
+*{
+  box-sizing: border-box;
+  margin:10px;
+}
 .App {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
 }
+.App .App-header{
+  font-size: 2.5rem;
```

Comment:
some places use rem, some places use px, please use the same one.
either rem or px.